media-gfx/viewnior/viewnior-1.7-r2.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit meson xdg-utils
+
+MY_PN=Viewnior
```

Comment:
Instead of this, you can use `${PN^}" in place. Or `${P^}". But up to preference, do as you like more.

Reply:
Thanks for the tip! done.